### PR TITLE
learn.jquery.com: remove templating for contributors attributions

### PR DESCRIPTION
**Pull request by arthurvr**
__Don't merge this yet, It must be merged the same time with a PR in the `learn.jquery.com` repo. Just open for review now.__

As discussed in https://github.com/jquery/learn.jquery.com/issues/602, this section is going to get replaced by a link to the Github contributor graphs.

Also: seems like my editor removed some trailing whitespace in the css file, don't care about it :beers:

### CODE REVIEW

File: themes/learn.jquery.com/style.css [hunk 1]
```diff
@@ -53,11 +53,11 @@ a {
...
-	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E)); 
-	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E); 
+	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E));
+	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E);
+	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E);
```
Comment (mzgol) [inline themes/learn.jquery.com/style.css]: Ouch, we don't have unprefixed gradients here; that's bad. Most of these prefixes are no longer needed btw.

cc @scottgonzalez

File: themes/learn.jquery.com/style.css [hunk 2]
```diff
@@ -53,11 +53,11 @@ a {
 	top: 11px;
 	width: 120px;
 	background-color: #ffea94;
-	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E)); 
-	background-image: -webkit-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:    -moz-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:     -ms-linear-gradient(top, #ffea94, #FFEA5E); 
-	background-image:      -o-linear-gradient(top, #ffea94, #FFEA5E); 
+	background-image: -webkit-gradient(linear, left top, left bottom, from(#ffea94), to(#FFEA5E));
```
Comment (mzgol) [inline themes/learn.jquery.com/style.css]: @arthurvr These changes don't belong in this PR; they don't correspond to the PR title.

File: themes/learn.jquery.com/style.css (hunk 1 above)
Comment (arthurvr) [inline themes/learn.jquery.com/style.css]: True. But that's not something for this PR ;-) The only reason this appears in the diff is because the removed trailing whitespace.

File: themes/learn.jquery.com/style.css (hunk 2 above)
Comment (arthurvr) [inline themes/learn.jquery.com/style.css]: Nothing really changed, but If you guys like them out, it was just an automatism of my editor.

Comment (mzgol) [inline themes/learn.jquery.com/style.css]: Commits are best focused on what they describe. These changes have nothing to do with "removing templating for contributors attributions" so they shouldn't be included.

BTW, some Git-aware editors allow to automatically strip trailing spaces only for changed lines.

Comment (arthurvr) [inline themes/learn.jquery.com/style.css]: However I still think it's kinda stupid, I'il amend the commit and swap those changes out.

File: themes/learn.jquery.com/style.css (hunk 1 above)
Comment (mzgol) [inline themes/learn.jquery.com/style.css]: I've created an issue for that so that we don't continue this unrelated discussion here. See #335.

**Comment by arthurvr** [comment]
@mzgol Updated my PR to include the trailing whitespace again.

**Comment by arthurvr** [comment]
According PR in `learn.jquery.com`: jquery/learn.jquery.com#603

**Comment by scottgonzalez** [comment]
Landed in 661749994919730a1a0e712abce229a61481a702, forgot to add the ref for this PR.
